src/actions/index.js

```diff
@@ -0,0 +1,36 @@
+import {FETCH_TREND, ADD_FAVORITE, DELETE_FAVORITE, FETCH_DETAILS, SEARCH_ON_WEB} from './types'
+
+export function fetchTrend () {
+  return ({
+    type: FETCH_TREND
+  })
+}
+
+export function addFavorite (favorite) {
+  return({
+    type: ADD_FAVORITE,
+    favorite
```

Comment:
All actions should have a `payload` key, and inside that key, you put the data that describes the event (in this example `favorite`).

Should look like this:

```js
export function addFavorite(favorite) {
  return ({
    type: ADD_FAVORITE,
    payload: {
      favorite,
    }
  })
}
```